base_export_manager/static/src/js/base_export_manager.js

```diff
@@ -6,6 +6,15 @@ odoo.define('base_export_manager.base_export_manager', function(require) {
     'use strict';
     
     var DataExport = require('web.DataExport');
+    var core = require('web.core');
+    var Model = require('web.DataModel');
+    var ListView = require('web.ListView');
+    var Sidebar = require('web.Sidebar');
+    var _t = core._t;
+    var _lt = core._lt;
```

Comment:
Model, ListView, Sidebar objects are declared in code. and core is also used to declare _t, _lt variables.